### ISSUE TITLE
losing pixels due to odd stride?

### ISSUE THREAD

**Issue by drproduck** [andrewjong/video-outfit-transfer#15]
In nets.py line 144:
` nn.Conv2d(in_feat, out_feat, 3, 2, 1),`

for size == 128, this combination of kernel_size=3, stride=2, padding=1 will skip the last row and column of the image (see note of [https://pytorch.org/docs/stable/nn.html#convolution-layers](url))

despite this, the img size correctly reduces to half (= 64), thus no error.

An easy fix is to use (4,2,1) which reduces size to half and uses all pixels
Another useful is (3,1,1) which gives the same input size.

Should we change this?

**Comment by drproduck** [comment]
It does not fix destabilization (duh)
